diff -urN pygame-1.3.org/config_unix.py pygame-1.3/config_unix.py
--- pygame-1.3.org/config_unix.py	Fri Jan 25 23:45:26 2002
+++ pygame-1.3/config_unix.py	Fri Jan 25 23:59:36 2002
@@ -50,7 +50,7 @@
     Dependency('FONT', 'SDL_ttf.h', 'libSDL_ttf.so', 'SDL_ttf'),
     Dependency('IMAGE', 'SDL_image.h', 'libSDL_image.so', 'SDL_image'),
     Dependency('MIXER', 'SDL_mixer.h', 'libSDL_mixer.so', 'SDL_mixer'),
-    Dependency('SMPEG', 'smpeg.h', 'libsmpeg.so', 'smpeg'),
+    Dependency('SMPEG', 'smpeg/smpeg.h', 'libsmpeg.so', 'smpeg'),
 ]
 
 
