--- pygame-1.9.6/buildconfig/config_unix.py.orig	2019-04-25 08:14:17.000000000 +0200
+++ pygame-1.9.6/buildconfig/config_unix.py	2024-09-26 17:35:08.789307494 +0200
@@ -149,7 +149,7 @@ def main(sdl2=False):
     #these get prefixes with '/usr' and '/usr/local' or the $LOCALBASE
     if sdl2:
         origincdirs = ['/include', '/include/SDL2']
-        origlibdirs = ['/lib','/lib64','/X11R6/lib',
+        origlibdirs = ['/lib','/lib64','/libx32',
                        '/lib/i386-linux-gnu', '/lib/x86_64-linux-gnu']
     else:
         origincdirs = ['/include', '/include/SDL', '/include/SDL']
